misk-crypto/src/main/kotlin/misk/crypto/KeyManager.kt

```diff
@@ -83,43 +69,23 @@ class MacKeyManager @Inject internal constructor(
  * ```
  */
 @Singleton
-class DigitalSignatureKeyManager @Inject internal constructor(
-  private val injector: Injector
-) {
-  private val signers: HashMap<String, DigitalSignature> = LinkedHashMap()
+class DigitalSignatureKeyManager @Inject internal constructor(injector: Injector)
+    : MappedKeyManager<DigitalSignature>(injector, DigitalSignature::class.java) {
 
-  internal operator fun set(name: String, signerAndVerifier: DigitalSignature) {
-    if (!signers.containsKey(name)) {
-      signers[name] = signerAndVerifier
-    }
-  }
+    fun getSigner(name: String): PublicKeySign? = this[name]?.signer
```

Comment:
You can remove the `?` here because if the key isn't found, an exception will be thrown